app/views/admin/cms/collections/_item_list.html.erb

```diff
@@ -3,11 +3,11 @@
     <div class="col-2"><%= item.uid %></div>
     <div class="col-4"><%= item.title %></div>
     <div class="col-2">
-      <%= image_tag(item.image.thumb) if item.image.present? %>
+      <%= image_tag(item.image.thumb) if item.image.url.present? %>
     </div>
 
     <div class="col-1">
-      <%= check_box_tag :transcript_url, 'uploaded', item.script_url.present? %>
+      <%= check_box_tag :transcript_url, 'uploaded', item.present? %>
```

Comment:
If there are no items (transcripts) for a collection this partial won't be displayed so there will be at least one item available here.